internal/brokers/examplebroker/broker.go

```diff
@@ -324,20 +328,34 @@ func getMfaModes(info sessionInfo, supportedModes map[string]map[string]string)
 	return mfaModes
 }
 
-func getPasswdResetModes(info sessionInfo, supportedModes map[string]map[string]string) map[string]map[string]string {
+func getPasswdResetModes(info sessionInfo, supportedUILayouts []map[string]string) map[string]map[string]string {
```

Comment:
indeed, this is more correct as potentially the supportedUILayouts could change between calls…

Reply:
This wasn't exactly an issue here, since we were getting the modes for the supportedModes list (which already looked at the supportedLayouts and provided the available modes). The issue is that we were showing the resetpassword modes to the first list as well...

Reply:
ack :)